src/desktop/apps/auction/__tests__/routes.test.js

```diff
@@ -6,165 +6,184 @@ import { fabricate } from 'antigravity'
-//     const auctionQueries = {
-//       sale: {
-//         id: 'foo',
-//         is_auction: true,
-//         auction_state: 'open',
-//         is_live_open: true
-//       },
-//       me: {
-//         bidders: [{
-//           id: 'me',
-//           qualified_for_bidding: false,
-//           sale: {
-//             id: 'foo'
-//           }
-//         }]
-//       }
-//     }
-
-//     rewire.__set__('metaphysics', sinon.stub().returns(Promise.resolve(auctionQueries)))
-//     await routes.redirectLive(req, res, next)
-//     res.redirect.called.should.not.be.ok()
-//     next.called.should.be.ok()
-//   })
-
-//   it('does not redirect if the auction is not live', async () => {
-//     const auctionQueries = {
-//       sale: {
-//         id: 'foo',
-//         is_auction: true,
-//         auction_state: 'open',
-//         is_live_open: false
-//       },
-//       me: {
-//         bidders: [{
-//           id: 'me',
-//           qualified_for_bidding: false,
-//           sale: {
-//             id: 'foo'
-//           }
-//         }]
-//       }
-//     }
-
-//     rewire.__set__('metaphysics', sinon.stub().returns(Promise.resolve(auctionQueries)))
-//     await routes.redirectLive(req, res, next)
-//     res.redirect.called.should.not.be.ok()
-//     next.called.should.be.ok()
-//   })
-// })
+// FIXME: Race conditions in tests here. See failing tests: https://circleci.com/gh/artsy/force/5461#tests/containers/2
+xdescribe('#index', () => {
+  let req
+  let res
+  let next
+
+  beforeEach(() => {
+    sinon.stub(Backbone, 'sync')
+    req = {
+      body: {},
+      params: { id: 'foobar' },
+      user: new CurrentUser(fabricate('user')),
+    }
+    res = {
+      app: { get: sinon.stub().returns('components/page') },
+      locals: { sd: {} },
+      redirect: sinon.stub(),
+      render: sinon.stub(),
+      send: sinon.stub(),
+    }
+    next = sinon.stub()
+  })
+
+  afterEach(() => {
+    Backbone.sync.restore()
+  })
+
+  it('renders the index with the correct variables', () => {
+    const auctionQueries = {
+      sale: {
+        id: 'foo',
+        is_auction: true,
+        auction_state: 'open',
+        is_live_open: true,
+      },
+      me: {
+        bidders: [
+          {
+            id: 'me',
+            qualified_for_bidding: true,
+            sale: {
+              id: 'foo',
+            },
+          },
+        ],
+      },
+    }
+
+    rewire.__set__(
+      'metaphysics',
+      sinon.stub().returns(Promise.resolve(auctionQueries))
+    )
+    rewire.__set__('renderLayout', () => '<html />')
+    rewire.__set__('actions', {
+      fetchArtworksByFollowedArtists: () => ({ type: 'GET_ARTWORKS_SUCCESS' }),
+      fetchArtworks: () => ({ type: 'GET_ARTWORKS_SUCCESS' }),
+    })
+
+    // FIXME: Need to write more robust output test
+    routes.index(req, res, next).then(() => {
+      res.send.args[0][0].should.eql('<html />')
+    })
+  })
+})
+
+xdescribe('#redirectLive', () => {
```

Comment:
I know we had looked at these on your local machine and thought we'd fixed the race condition-- is that just pending an update to this branch?

Reply:
looks like it is. I'll push in the morning.